src/main/java/com/stripe/exception/APIException.java

```diff
@@ -12,7 +12,8 @@ public APIException(String message, String requestId, Integer statusCode, Throwa
     this(message, requestId, null, statusCode, e);
   }
 
-  public APIException(String message, String requestId, String code, Integer statusCode, Throwable e) {
+  public APIException(String message, String requestId, String code, Integer statusCode,
+      Throwable e) {
```

Comment:
are we sure this is better that one param per line? I'm not familiar with Java styling so wanted to ask

Reply:
The [style guide](https://google.github.io/styleguide/javaguide.html) doesn't enforce a specific style for how to line-wrap long declarations, but taking a look at Gson's source code, they use the same style of wrapping only when necessary: https://github.com/google/gson/blob/master/gson/src/main/java/com/google/gson/reflect/TypeToken.java#L185-L186.